jina/clients/request/helper.py

```diff
@@ -46,6 +47,14 @@ def _new_request_from_batch(_kwargs, batch, data_type, mode, queryset):
             _add_docs_groundtruths(req, batch, data_type, _kwargs)
         elif mode == RequestType.DELETE:
             _add_ids(req, batch)
+        elif mode == RequestType.RELOAD:
+            req.path = batch
+            req.workspace = kwargs.get('workspace')
```

Comment:
This will be changed once we have Florian's changes in #2257 . Then the DUMP request will be passed as a control request directly to the Pod. This is kept here to show that the Driver/Indexer combination works, as this was the way we did Requests up until now.